src/sotauptaneclient.cc

```diff
@@ -119,8 +119,7 @@ data::InstallOutcome SotaUptaneClient::OstreeInstall(const Uptane::Target &targe
   }
 }
 
-Json::Value SotaUptaneClient::OstreeInstallAndManifest(const Uptane::Target &target) {
-  Json::Value operation_result;
+void SotaUptaneClient::OstreeInstallAndManifest(const Uptane::Target &target) {
```

Comment:
This function name seems like a misnomer at this point. This is just like a pre-install step or status check before and after `OstreeInstall` now.

Reply:
It mostly sets operation result now. `OstreeInstallSetResult`?

Reply:
Sure. Or maybe `SetOstreeInstallResult` to put the verb first?

Reply:
Well, it doesn't just set result, it makes OstreeInstall first.